### PR TITLE
Create agents management menu and move sections

### LINKED ISSUE #7069: Create agents management menu
# Description

We want to create a new left menu section called

"Agents management"

Which will be the first item in the "* management" menu entries.

- [x] Move "Endpoint Summary" from "Server management" to "Agents management", and change its name to "Summary"

- [x] Move "Groups" from "Server management" to "Agents management", and preserve its name.

- [x] Show the Dashboard management section first and then the Indexer management section.

### PR DESCRIPTION
**Pull request by chantal-kelm**
### Description

Create an Agents management menu and move sections:

Endpoint Summary -> Change its name to Summary
Endpoint Groups
 
### Issues Resolved

#7069 

### Evidence

image

### Test

Verify that the new Agent management section exists and that it contains the following sections inside: 

- Summary

- Endpoint Groups

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: plugins/main/public/utils/applications.ts [hunk 1]
```diff
@@ -897,12 +897,20 @@ export const Categories = [
...
+    order: 600,
+    euiIconType: 'indexRollupApp',
+  },
   {
     id: 'wz-category-server-management',
     label: i18n.translate('wz-app-category-server-management', {
       defaultMessage: 'Server management',
     }),
-    order: 600,
+    order: 700,
```
Comment (Desvelao) [inline plugins/main/public/utils/applications.ts:913]: **nitpick**: the `order` of some categories were changed. According to the application order convention (https://github.com/wazuh/wazuh-dashboard-plugins/blob/1e106ace081d9821ad1b350dfb1a9214c1bd2412/plugins/main/public/utils/applications.ts#L10-L30), they should have an `order` related to the category they belong to.

Comment (chantal-kelm) [reply plugins/main/public/utils/applications.ts:914]: Fixed in [commit]([4c7b306](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7112/commits/4c7b3060debaeacbd35a9d64a76e3ea11b73183c))

**Review by yenienserrano — APPROVED** [review]
LGTM

image

**Comment by Desvelao** [comment]
# :green_apple: Test

image

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**